### PR TITLE
gnrc 6lowpan: minor fixes

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c [hunk 1]
```diff
@@ -216,7 +214,10 @@ static void _send(gnrc_pktsnip_t *pkt)
 
     hdr = pkt2->data;
     iface = gnrc_sixlowpan_netif_get(hdr->if_pid);
-    datagram_size = gnrc_pkt_len(pkt2->next);
+#ifdef MODULE_GNRC_SIXLOWPAN_FRAG
+    /* datagram_size: pure IPv6 packet without 6LoWPAN dispatches or compression */
+    size_t datagram_size = gnrc_pkt_len(pkt2->next);
```
Comment (authmillenon) [inline sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c]: Please replace this variable wit the function call in https://github.com/RIOT-OS/RIOT/pull/3974/files#diff-e87e12a912a7eb2cbb8a16f4ebeccabaR281 (and other places I might have overlooked).

**Comment by authmillenon** [comment]
ACK apart from comments

File: sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c (hunk 1 above)
Comment (OlegHahm) [inline sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c]: Found a better solution.

**Comment by authmillenon** [comment]
Can't test this WE. If it works my, I trust you and say ACK, but please double check. (with gnrc_sixlowpan_frag included and not).

**Comment by OlegHahm** [comment]
Will test.

**Comment by OlegHahm** [comment]
Ok to squash?

**Comment by authmillenon** [comment]
Yes.

**Comment by OlegHahm** [comment]
squashed

**Comment by OlegHahm** [comment]
Did build and test in both configurations. Still working.

**Pull request merged**